blog/boa-release-21/index.md

```diff
@@ -0,0 +1,627 @@
+---
+layout: post
+tags: [post]
+title: "Boa release v0.21"
+description: "Boa release v0.21"
+authors: boa-dev
+---
+
+## Summary
+
+Boa v0.21 is now available! After 9 months of development we are very
+happy to present you the latest release of the Boa JavaScript engine.
+Boa makes it easy to embed a JS engine in your projects, and you can
+even use it from WebAssembly. See the [about](/about) page for more
+info.
+
+In this release, our conformance has grown from 89.92% to 94.12% in the
+official ECMAScript Test Suite (Test262). Our growth in conformance is
+driven by increased conformance for Temporal (discussed further below)
+with the rest of the development effort being focused on performance,
+internal improvements, and runtime features. We will continue to implement
+more of the specification; however, as of the current moment, Boa's conformance
+aligns with the major browser engine's conformance, so future increases
+in conformance will be minor or bound to the feature size going forward.
+
+You can check the full list of changes [here][changelog], and the full
+information on conformance [here][conformance].
+
```

Comment:
We should add the changelog and conformance links here, so that we don't break the extract:
```suggestion

[changelog]: https://github.com/boa-dev/boa/blob/v0.21/CHANGELOG.md
[conformance]: https://boajs.dev/boa/test262/

```